src/index.ts

```diff
@@ -119,11 +120,15 @@ export class Client {
     return this.unary(API.ModelHas, req) as Promise<ModelHasReply.AsObject>
   }
 
-  public async modelFind(storeID: string, modelName: string) {
+  public async modelFind(storeID: string, modelName: string, query: JSONQuery) {
     const req = new ModelFindRequest()
     req.setStoreid(storeID)
     req.setModelname(modelName)
-    return this.unary(API.ModelFind, req) as Promise<ModelFindReply.AsObject>
+    req.setQueryjson(Buffer.from(JSON.stringify(query)))
+    const res = await this.unary(API.ModelFind, req) as ModelFindReply.AsObject
+    // @todo: Do we want to do this? Otherwise, the caller has to decode the base64 string...
+    res.entitiesList = res.entitiesList.map(entity => Buffer.from(entity as string, 'base64').toString())
```

Comment:
There are nicer ways to decode base64 strings that don't require Buffers, so if this is what we wanna do, let's use one. Otherwise, we're leaving it up to the caller to decode etc, which maybe isn't as nice? Having said that, this whole gRPC wrapper lib is pretty low-level, so maybe we should leave the niceties to a higher level lib anyway?